section/definitions.md

```diff
@@ -51,6 +51,11 @@ Furthermore, we define following concepts:
   _The ontology term is defined at [fno:Implementation](#fno-implementation)._
 - A <dfn>mapping</dfn> links (abstract) <a data-lt="function">functions</a> and (concrete) <a data-lt="implementation">implementations</a>.
   _The ontology term is defined at [fno:Mapping](#fno-Mapping)._
+- A <dfn>composition</dfn> combines multiple functions by mapping <a data-lt="parameter">parameters</a> or <a data-lt="output">outputs</a>  
+of <a data-lt="function">constituent functions</a> onto one another, defining a nesting or concatenation of functions.
+  _The ontology term is defined at [fnoc:Composition](#fnoc-Composition). The composition consists of a set of CompositionMappings.
```

Comment:
I think a `_` is missing after [fnoc:Composition]

Reply:
done